lib/hardware_services/inventory_drivers/MockInventoryDriver.py

```diff
@@ -151,14 +151,47 @@ def load_data(self, quadsinstance, force):
             quadsinstance.logger.error(ex)
             exit(1)
 
-    def write_data(self, quadsinstance, doexit = True):
-        quadsinstance.quads_write_data_(doexit)
+    def write_data(self, quadsinstance, doexit):
+        try:
+            stream = open(quadsinstance.config, 'w')
```

Comment:
Yeah. This piece is the original Quads code. I just moved it from one place to another.